src/core/proto/mem_buf_desc.h

```diff
@@ -127,6 +129,21 @@ class mem_buf_desc_t {
         return lwip_pbuf.ref;
     }
 
+    /*
```

Comment:
Please mention in the comment that this is related to XLIO API

Reply:
Done.